src/helpers/fromHex.js

```diff
@@ -1,19 +1,19 @@
-import { toUtf8, toDecimal, toAscii } from 'web3-utils'
+import { ethers, BigNumber } from 'ethers'
+
 export default () =>
   /**
    * Returns the string representation of a given hex value
    *
    * @param {string} hex The hex string
-   * @param {string} [to='utf8'] The type to convert the hex from (supported types: 'utf8', 'number', 'decimal')
+   * @param {string} [to='utf8'] The type to convert the hex from (supported types: 'utf8', 'bytes32', 'number')
    * @return {radspec/evaluator/TypedValue}
    */
-  (hex, to = 'utf8') =>
-    ({
-      type: 'string',
-      value:
-        to === 'utf8'
-          ? toUtf8(hex)
-          : to === 'ascii'
```

Comment:
The support of `ascii` is the only one that we couldn't keep backward compatible, instead, I include support for [`bytes32`strings](https://docs-beta.ethers.io/api/utils/strings/#Bytes32String). 

Given the above, the refactoring will be a major release.